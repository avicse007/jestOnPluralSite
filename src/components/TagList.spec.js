import React from "react";
import TagList from "./TagsList";
import rendrer from "react-test-renderer";

describe('The tagList', ()=>{
    it('render as expected',()=>{
        const tree = rendrer
        .create(<TagList tags ={['css','html','shift']}/>)
        .toJSON();
        console.log(tree);
        expect(tree).toMatchSnapshot();

    });
} );