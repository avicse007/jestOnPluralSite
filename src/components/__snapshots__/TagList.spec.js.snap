// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The tagList render as expected 1`] = `
<div>
  <code>
    css
  </code>
  <code>
    html
  </code>
  <code>
    shift
  </code>
</div>
`;
